package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] As the previous reviewer has requested, kindly ensure that you add a script to run your webpack server in the package,json file. For example, you can add something like
``` js
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "watch": "webpack --watch",
    "start": "webpack serve --open",
    "build": "webpack"
  },
```